### DISCUSSION TITLE
Section Page Numbering does not start at 1

### DISCUSSION THREAD

**Discussion by rima1005** [QuestPDF/QuestPDF#708] [Q&A] [answered]
Hi, I want to create a document with some reusable content, therefore i have the following setup:

```csharp
        container.Page(page =>
        {
            page.Size(PageSizes.A4);
            page.Content().Section("section1").Element(ComposeContent);
            page.Header().Element(ComposeHeaderWithSection("section1"));
        });

        container.Page(page =>
        {
            page.Size(PageSizes.A4);
            page.Content().Element(SomeOtherContent);
        });
        container.Page(page =>
        {
            page.Size(PageSizes.A4);
            page.Content().Section("section2").Element(ComposeContent);
            page.Header().Element(ComposeHeaderWithSection("section2"));
        });

```

in the ComposeHeaderWithSection(string section) i print the PageNumberWithinSection(string section).
So in general, the behaviour is like i expected. The "section2" should act like a copy and therefore the page numbers are the same as in "section1" but somehow, both section start with page number 2 instead of 1 and i don't get why it does not start at 1? Are there any ideas what i am doing wrong?

**Comment by MarcinZiabek** [comment]
Are section names unique across the document? What API do you use to show the page number within section? What version of the library?

**Reply by rima1005** [reply, marked as answer]
Hi, it seems the latest release (2023.10.2) from yesterday just resolved this issue. Thank you!

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
